docker-stack.yml

```diff
@@ -0,0 +1,373 @@
+version: '3.7'
```

Comment:
This seems like an accidental check-in.

Reply:
It was intended

Reply:
How does this relate to retrying HTTP requests?

Reply:
this is the current host configuration. I'm just being lazy and commiting it with this change.

Reply:
Indeed.